<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>366</width>
      <height>588</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_robots">
     <item>
      <widget class="QGroupBox" name="Ro_box">
       <property name="title">
        <string>Robot opérateur (orange)</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="Ro_layout">
          <item>
           <layout class="QVBoxLayout" name="Ro_layout_boutons">
            <item>
             <widget class="QPushButton" name="Ro_bouton_demarrer">
              <property name="text">
               <string>Démarrer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Ro_bouton_calibrer">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Calibrer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Ro_bouton_connecter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Connecter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Ro_bouton_identifier">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Identifier</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Ro_layout_etat">
            <item>
             <widget class="QGroupBox" name="Ro_box_q">
              <property name="title">
               <string>Articulations</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="Ro_layout_q">
                 <item>
                  <layout class="QHBoxLayout" name="Ro_layout_q1">
                   <item>
                    <widget class="QLabel" name="Ro_label_q1">
                     <property name="text">
                      <string>q1 [rad] :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="Ro_horizontalSpacer_q1">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Ro_q1">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="Ro_layout_q2">
                   <item>
                    <widget class="QLabel" name="Ro_label_q2">
                     <property name="text">
                      <string>q2 [rad] :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="Ro_horizontalSpacer_q2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Ro_q2">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="Ro_box_pos">
              <property name="title">
               <string>Position</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QVBoxLayout" name="Ro_layout_pos">
                 <item>
                  <layout class="QHBoxLayout" name="Ro_layout_x">
                   <item>
                    <widget class="QLabel" name="Ro_label_x">
                     <property name="text">
                      <string>x [mm] :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="Ro_horizontalSpacer_x">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Ro_x">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="Ro_layout_y">
                   <item>
                    <widget class="QLabel" name="Ro_label_y">
                     <property name="text">
                      <string>y [mm] :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="Ro_horizontalSpacer_y">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Ro_y">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="Rt_box">
       <property name="title">
        <string>Robot téléopéré (bleu)</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="Rt_layout">
          <item>
           <layout class="QVBoxLayout" name="Rt_layout_boutons">
            <item>
             <widget class="QPushButton" name="Rt_bouton_demarrer">
              <property name="text">
               <string>Démarrer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Rt_bouton_calibrer">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Calibrer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Rt_bouton_connecter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Connecter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Rt_bouton_identifier">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Identifier</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Rt_layout_etat">
            <item>
             <widget class="QGroupBox" name="Rt_box_q">
              <property name="title">
               <string>Articulations</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QVBoxLayout" name="Rt_layout_q">
                 <item>
                  <layout class="QHBoxLayout" name="Rt_layout_q1">
                   <item>
                    <widget class="QLabel" name="Rt_label_q1">
                     <property name="text">
                      <string>q1 [rad] :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="Rt_horizontalSpacer_q1">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Rt_q1">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="Rt_layout_q2">
                   <item>
                    <widget class="QLabel" name="Rt_label_q2">
                     <property name="text">
                      <string>q2 [rad] :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="Rt_horizontalSpacer_q2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Rt_q2">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="Rt_box_pos">
              <property name="title">
               <string>Position</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="Rt_layout_pos">
                 <item>
                  <layout class="QHBoxLayout" name="Rt_layout_x">
                   <item>
                    <widget class="QLabel" name="Rt_label_x">
                     <property name="text">
                      <string>x [mm] :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="Rt_horizontalSpacer_x">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Rt_x">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="Rt_layout_y">
                   <item>
                    <widget class="QLabel" name="Rt_label_y">
                     <property name="text">
                      <string>y [mm] :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="Rt_horizontalSpacer_y">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="Rt_y">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="figure" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>401</width>
      <height>173</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="teleop_layout">
     <item>
      <widget class="QLabel" name="teleop_label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Système de téléopération</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="teleop_layout_boutons">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="teleop_bouton_identifier">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Identifier</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="teleop_bouton_lancer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Lancer/Arrêter</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="teleop_box_ylim">
       <property name="title">
        <string>Limite de l'espace</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="teleop_layout_ylim">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="teleop_label_ylim">
            <property name="text">
             <string>ylim [mm] :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="teleop_ylim">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>-300.000000000000000</double>
            </property>
            <property name="maximum">
             <double>-150.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>-250.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Ro_bouton_demarrer</sender>
   <signal>clicked(bool)</signal>
   <receiver>Ro_bouton_calibrer</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ro_bouton_calibrer</sender>
   <signal>clicked(bool)</signal>
   <receiver>Ro_bouton_connecter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Rt_bouton_demarrer</sender>
   <signal>clicked(bool)</signal>
   <receiver>Rt_bouton_calibrer</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Rt_bouton_calibrer</sender>
   <signal>clicked(bool)</signal>
   <receiver>Rt_bouton_connecter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teleop_bouton_identifier</sender>
   <signal>toggled(bool)</signal>
   <receiver>teleop_bouton_lancer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
